### ISSUE TITLE
dead link to video in readme

### ISSUE THREAD

**Issue by dnault** [couchbaselabs/CouchVersion#9]
Hi @deniswsrosa . Looks like this link in the readme might be dead?

https://share.vidyard.com/watch/98GgKFiyi4tz2WuLQVVfzW

**Comment by paulvi** [comment]
yep, and it looks like auther does not care much to looks through issues at least once in month.
